src/pages/Calculator/Steps/SeedingMethod/index.js

```diff
@@ -15,11 +15,22 @@ import Dropdown from '../../../../components/Dropdown';
 import '../steps.scss';
 import { setOptionRedux, setSeedingMethodsRedux } from '../../../../features/calculatorSlice/actions';
 
+// styles for left grid
+const FullGrid = styled(Grid)(() => ({
+  '&.MuiGrid-item': {
+    // border: '1px solid #c7c7c7',
+    // borderRight: 'none',
+    boxShadow: '1px 1px 10px 1px lightgrey',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  },
+}));
 // styles for left grid
 const LeftGrid = styled(Grid)(() => ({
```

Comment:
And also remove the commented code if they are not needed anymore.

Reply:
> And also remove the commented code if they are not needed anymore.

This is my fault i asked him to leave it in till i had a chance to review things since it was his first ticket.  But agreed they can be removed now.